app/templates/components/ccg-search-result.hbs

```diff
@@ -1,3 +1,4 @@
+<a href="http://govwizely.github.io/explorer/#/country-commercial-guides" style="float:right;">New Search</a>
```

Comment:
When the ITA fork gets this code their version is going to link back to GovWizely's.